src/expr/src/relation/mod.rs

```diff
@@ -3203,6 +3204,30 @@ impl RustType<proto_window_frame::ProtoWindowFrameBound> for WindowFrameBound {
     }
 }
 
+/// Maximum iterations for a LetRec.
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub struct LetRecLimit {
+    /// Maximum number of iterations to evaluate.
+    pub max_iters: NonZeroU64,
+    /// Whether to throw an error when reaching the above limit.
+    /// If false, we simply use the current contents of each Id as the final result.
+    pub error_at_limit: bool,
+}
+
+/// The default value of `LetRecLimit::error_at_limit` when using the RECURSION LIMIT option of WMR
+/// without ERROR AT or RETURN AT.
+pub const WMR_ERROR_AT_LIMIT_DEFAULT: bool = true;
+
+impl Display for LetRecLimit {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "[recursion_limit={}", self.max_iters)?;
+        if self.error_at_limit != WMR_ERROR_AT_LIMIT_DEFAULT {
+            write!(f, ", error_at_limit={}", self.error_at_limit)?;
```

Comment:
nit: using `error_at_limit` means that we will print `error_at_limit=true` most of the time.
If we use a boolean flag that denotes the inverse (like `return_at_limit`) we can only print `return_at_limit` if the user has requested the non-default behavior, which I assume would lead to better plans.

Reply:
I changed from `error_at_limit` to `return_at_limit`, and changed the printing to not print the `=true`.